mypy/checkmember.py

```diff
@@ -32,9 +32,9 @@ def analyze_member_access(name: str,
                           is_operator: bool,
                           builtin_type: Callable[[str], Instance],
                           not_ready_callback: Callable[[str, Context], None],
-                          msg: MessageBuilder,
+                          msg: MessageBuilder, *,
+                          original_type: Type,
```

Comment:
Now that you've made this mandatory and passed it explicitly everywhere, do you still need line 53 below? (`original_type = original_type or typ` -- I can't put a comment there.)